cmd/dagger/cmd/project/update.go

```diff
@@ -34,7 +34,9 @@ var updateCmd = &cobra.Command{
 
 		if len(args) == 0 {
 			lg.Debug().Msg("No package specified, updating all packages")
-			pkg.Vendor(ctx, cueModPath)
+			if err := pkg.Vendor(ctx, cueModPath); err != nil {
+				lg.Fatal().Err(err).Msg("vendor pkg failed")
```

Comment:
When looking at the other log messages I noticed they all have "packages" spelled out instead of "pkg".
I would suggest sticking to the same pattern.

Reply:
Actually, I think this is going to be [addressed here](https://github.com/dagger/dagger/pull/2105/files?diff=split&w=0#diff-70e7d3f26a9dd1cde8b2cfec273df6d61233d5c098b2bdcf83501cf232a12004L37).